src/bindings/capi/tvgCapi.cpp

```diff
@@ -254,6 +254,21 @@ TVG_EXPORT Tvg_Result tvg_shape_append_path(Tvg_Paint* paint, const Tvg_Path_Com
     return (Tvg_Result) reinterpret_cast<Shape*>(paint)->appendPath((PathCommand*)cmds, cmdCnt, (Point*)pts, ptsCnt);
 }
 
+TVG_EXPORT Tvg_Result tvg_shape_get_path_points(const Tvg_Paint* paint, const Tvg_Point** pts, uint32_t* cnt)
+{
+    if (!paint || !pts || !cnt) return TVG_RESULT_INVALID_ARGUMENT;
```

Comment:
Please suggest to return valid values.
auto ret = reinterpret_cast<const Shape*>(paint)->pathCoords((const Point**)pts);   //pts null check is inside performed.
if (cnt) *cnt = ret;